types/mem-fs-editor/index.d.ts

```diff
@@ -30,6 +31,8 @@ export interface Editor {
 
     append(filepath: string, contents: WriteContents, options?: AppendOptions): string;
 
+    appendTpl(filepath: string, contents: WriteContents, context?: TemplateData, templateOptions?: TemplateOptions, options?: AppendOptions): void;
```

Comment:
Unlike `append`,  `appendTpl` don't have `create` in the `options` object. Could you create a `AppendTplOptions` interface for it?

Reply:
I'm sorry, I don't understand.
As far as I can see `appendTpl` uses exactly the same `options` object as `append` (it acts as an overload and passes the object to `append`).
Also, I don't see `create` anywhere in the options object as it is used by `append`.